### PR TITLE
Languages no longer have a color

### PR DESCRIPTION
**Pull request by lzimann**
None of the maintainers seem to like these having a color. Plus, adding colors to regular chat is awful and it should keep being for radio/hiveminds only.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Where was the idea thread or any mention of this in the forums or coderbus? Just random creature comfort feature that you see  on other servers.

**Comment by duncathan** [comment]
since when do PRs have to pass forum approval
I don't much care about this one in particular but like that's not a thing is it

**Comment by coiax** [comment]
How are you supposed to tell languages apart? You need to provide some mechanic that people can tell which language you are speaking in, otherwise there's little point in the difference between this and the old system.

**Comment by RemieRichards** [comment]
```
Some Fucko: [,0] Radio: I am speaking in Galactic Common!
Some Fucko: [,r]: FOR ENGINE! //so engine -> ratvar :3
```

**Comment by duncathan** [comment]
remie I'll be honest I have no clue what your message here means LOL

**Comment by KorPhaeron** [comment]
Coiax has a good point

**Comment by RemieRichards** [comment]
@duncathan languages are like radios, they have keys in chat, you say ```,SOME_KEY SOME TEXT``` to speak in that language, default is Galactic Common (or ```,0```) but there's ```,r``` for ratvar, etc.

**Comment by coiax** [comment]
@RemieRichards you would have to show that for all listeners, not just speaker and observer. Otherwise how do I know if my lizard metabuddy is shouting DEATH TO ALL HUMANS in Lizardspeak or Galcom.

**Comment by RemieRichards** [comment]
@coiax I mean sure, I didn't consider them understanding it (because not understanding is covered by the scramble text)

**Comment by KorPhaeron** [comment]
Someone has to code said replacement before this can get merged then

**Comment by KorPhaeron** [comment]
I agree colours are not ideal though

**Comment by coiax** [comment]
@KorPhaeron :sagittarius: including little emojis/icons/images at the start of the sentence would be okay, we'd need some pretty icons (and as @RemieRichards said, hovertext if possible), but I'm sure @WJohn could design pretty ones.

**Comment by Fox-McCloud** [comment]
There needs to be some way to discern if a language is galatic common or not; if you were to be employed in a workplace with a particular set of individuals that spoke certain distinct languages, you're going to quickly pick up what language is what, even if you have zero ideas of what they're saying, based purely on how it sounds.

There's not a whole lot of ways you can get this across; coloring is probably the easiest; an emoji is another.

I dislike the idea of just a simple `[x]` format though; it's easy to miss, at a glance, despite the fact this is something that should probably be pretty boldly different.

**Comment by RemieRichards** [comment]
@Fox-McCloud as @coiax just said, they don't have to be as abstract, they can be little icons (still with the hoverable expansion)

**Comment by coiax** [comment]
Remember we can also adjust font sizes and types, we can attach arbitrary spans. We could additionally change the priority of speech verbs, so use of a language required you to use that verb. But then people would start debating about the relative priority of lizard hissing...

**Comment by RemieRichards** [comment]
(icons || bracket keys) with hovertext is the way to go, imo.

File: code/modules/language/ratvar.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	exclaim_verb = "clanks"
 	key = "r"
 	default_priority = 10
-	spans = list(SPAN_ROBOT, "brass")
```
Comment (Cyberboss) [inline code/modules/language/ratvar.dm]: This is changing preexisting functionality though

File: code/modules/language/xenocommon.dm [hunk 2]
```diff
@@ -4,7 +4,6 @@
 	speech_verb = "hisses"
 	ask_verb = "hisses"
 	exclaim_verb = "hisses"
-	spans = list("alien")
```
Comment (Cyberboss) [inline code/modules/language/xenocommon.dm]: Same here

**Comment by XDTM** [comment]
I think so as well, ideally with separate icons for speakers and non-speakers (for example, drone, swarmer and robotic will all have the same icon for nonspeakers, but specific ones for speakers), so you can have "disguised" languages.

**Comment by coiax** [comment]
@XDTM I don't know what this is about disguised languages, drone, machine and swarmer all sound different to non-speakers.

**Comment by XDTM** [comment]
I wanted to add syndicate codespeak, that would sound like random words in common to nonspeakers, and having a syndicate icon to it would be a bit of a tell. Then again i guess i could just snowflake it or something.

**Comment by coiax** [comment]
@XDTM you're making a language that consists of "ATMOSPHERIC TECH. CUBRE LIBRE. DARK GODS. MODESTY", and think people won't notice? :P

**Comment by XDTM** [comment]
It leaves a tiny bit of doubt that it might be just someone fooling around, like toy eswords

**Comment by Jordie0608** [comment]
In the absence of hovertext and whatnot, how about just
**Griff Spaceman** says in Ratvarian, "Quack"
`[person] [verb]s in [language], "[message]"`

**Comment by dannno** [comment]
if languages mask themselves in unique ways or even just use different fonts (not sure how many are at our disposal) there's not need for colors

**Comment by coiax** [comment]
Remember, even people who understand the language need to know which language was spoken.

**Comment by TalkingCactus** [comment]
Do fonts instead.

**Comment by ExcessiveUseOfCobblestone** [comment]
@iamgoofball time to RS text

**Comment by Iamgoofball** [comment]
:^q   p
     w

**Comment by Iamgoofball** [comment]
>>>

**Comment by Jordie0608** [comment]
I think it's trying to communicate.

**Comment by JJRcop** [comment]
flash2: wave2: scroll: He's just beginning to learn the language, give him time

**Comment by KorPhaeron** [comment]
Stale

**Pull request closed without merging**